apps/production/utils.py

```
@@ -163,6 +164,10 @@ def _get_stock_info(ingredient_id):
         for order in orders:
             if order.id in replaced_ids:
                 continue
+            # Polévka se na papírovou výdejku netiskne – kuchař ji zapisuje až
```

Ruff (0.15.21)

[warning] Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)